pkg/asset/agent/image/ignition.go

```diff
@@ -707,3 +714,28 @@ func getPublicContainerRegistries(registriesConfig *mirror.RegistriesConf) strin
 
 	return "quay.io"
 }
+
+func addNTPSources(config *igntypes.Config, infraEnv *v1beta1.InfraEnv) error {
```

Comment:
I wonder if we should make a public function somewhere for this.
This is already duplicating https://github.com/openshift/installer/blob/master/pkg/asset/imagebased/configimage/clusterconfiguration.go#L210-L216 and in a short time it will have to be, uh, triplicated for METAL-1136.

Reply:
:thinking: Found in https://github.com/openshift/assisted-service/blob/master/internal/ignition/templates/add-ntp-sources.sh (and was also in https://github.com/openshift/installer/blob/6320ceb8f5c0ca351f16924e7888088fa97eb64b/pkg/asset/imagebased/configimage/clusterconfiguration.go#L210). Probably we could make it more shared _at least_ within the installer repo, as a starting point.